### PR TITLE
added nigeria

### PR DESCRIPTION
**Pull request by michaelchukwu**

### CODE REVIEW

**Comment by michaelchukwu** [comment]
added the Nigerian map

**Comment by VictorCazanave** [comment]
Thanks a lot for you contribution @michaelchukwu ! 😃

File: packages/nigeria/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# 🇳🇬 Nigeria
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all 36 states of the Federal Republic of Nigeria(including the FCT):
```
Comment (VictorCazanave) [inline packages/nigeria/README.md:5]: To be consistent with other maps: `Contains all the states and the Federal Capital Territory (FCT) of Nigeria:`

File: packages/nigeria/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* Osun
+* Oyo
+* Plataeu
+* Rivers
+* Sokoto
+* Taraba
+* Yobe
+* Zamfara
+
+
```
Comment (VictorCazanave) [inline packages/nigeria/README.md:44]: Could you remove the extra empty line please? 😄

File: packages/nigeria/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+The original map is available [here](https://mapsvg.com/maps/nigeria) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Add viewBox
+* Rename ids
+* Sort `<path/>` alphabetically
```
Comment (VictorCazanave) [inline packages/nigeria/README.md:68]: Could you list on the real changes please?

File: packages/nigeria/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# 🇳🇬 Nigeria
```
Comment (VictorCazanave) [inline packages/nigeria/README.md:1]: Could you also add it in the main `README.md` please?

File: packages/nigeria/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	"keywords": [
+		"svg",
+		"map",
+		"nigeria"
+	],
+	"author": "Michael Chukwu <naemugo@gmail.com>",
+	"publishConfig": {
+		"access": "public"
+	},
+	"license": "CC-BY-4.0"
```
Comment (VictorCazanave) [inline packages/nigeria/package.json:19]: Could you add the `LICENSE.md` please?

File: packages/nigeria/nigeria.svg [hunk 6]
```diff
@@ -0,0 +1,155 @@
+<svg
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="2.667421 13.892750 14.680752 4.269658"
```
[the remark this answers is not in the document]
Comment (michaelchukwu) [reply packages/nigeria/nigeria.svg:3]: alright done

**Comment by VictorCazanave** [comment]
Thanks you 👍 

@all-contributors please add @michaelchukwu for code

**Pull request merged**